groups/rds/data.tf

```diff
@@ -0,0 +1,63 @@
+data "vault_generic_secret" "account_ids" {
+  path = "aws-accounts/account-ids"
+}
+
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.environment}/${var.repository_name}"
+}
```

Comment:
See: https://github.com/companieshouse/ci-grafana-terraform/pull/1/files#r1301125858

Reply:
Which path should be used, please?